Chap_API_Server.tex

```diff
@@ -185,18 +182,13 @@ \subsection{\code{PMIx_server_register_nspace}}
             \item \pastePRRTEAttributeItem{PMIX_SERVER_NSPACE}
             \item \pastePRRTEAttributeItem{PMIX_SERVER_RANK}
             \item \pastePRRTEAttributeItem{PMIX_NPROC_OFFSET}
-            \item \pastePRRTEAttributeItem{PMIX_APPLDR}
             \item \pastePRRTEAttributeItem{PMIX_SESSION_ID}
             \item \pastePRRTEAttributeItem{PMIX_ALLOCATED_NODELIST}
             \item \pastePRRTEAttributeItem{PMIX_JOB_NUM_APPS}
             \item \pastePRRTEAttributeItem{PMIX_MAPBY}
             \item \pastePRRTEAttributeItem{PMIX_RANKBY}
             \item \pastePRRTEAttributeItem{PMIX_BINDTO}
         \end{itemize}
-    \item for each application in the given namespace:
-        \begin{itemize}
-            \item \pastePRRTEAttributeItem{PMIX_APP_SIZE}
```

Comment:
`app_size` was also mentioned in the required section.

Reply:
Now thinking of that I figured out that it might be related to the case of single-application where RM is still allowed to publish this info key.
If this is the case I can revert this change.

Reply:
I should think "app_size" would be required.

Reply:
Meaning you agree that we can remove it here?
But what if it is a single application job. According to the standard RM is not required to post application keys, but I assume it still can do it if it wants to.

Reply:
Meaning that app_size is required, but I need to look at the total section and not just these commit snippets to see how to organize it. 😄

Reply:
Here I think I was deleting app_size from the "optional" subgroup.

Reply:
> Meaning that app_size is required, but I need to look at the total section and not just these commit snippets to see how to organize it. 

You have one negative from me now.